picard/ui/options/tags.py

```diff
@@ -151,8 +152,9 @@ def completer_activated(self, text):
         current = input_field.text()
         i = input_field.cursorPosition()
         p = len(self.completer.completionPrefix())
-        input_field.setText("%s%s %s" % (current[:i - p], text, current[i:]))
-        input_field.setCursorPosition(i - p + len(text) + 1)
+        start_text = current[:i - p].rstrip()
```

Comment:
I know that this isn't your code, but would it be possible to rename these (i, p) to something that explains what's happening here?